harness/nbjunit/src/org/netbeans/junit/NbTestCase.java

```diff
@@ -739,7 +739,7 @@ static public void assertFile(String message, File test, File pass, File diff, D
         } else {
             try {
                 if (diffImpl.diff(test, pass, diffFile)) {
-                    throw new AssertionFileFailedError(message, null == diffFile ? "" : diffFile.getAbsolutePath());
+                    throw new AssertionFileFailedError(message+"\n diff: "+diffFile, null == diffFile ? "" : diffFile.getAbsolutePath());
```

Comment:
How large are the files? I think the `diffFile` should lenght limited.

Reply:
it only prints the fileneme. so that you can quickly cat in bash to see what the problem is.

Reply:
A should have been obvious - thx.